src/components/ui/Slider.tsx

```
@@ -3,23 +3,29 @@ import { View, PanResponder, ViewProps, StyleSheet, Dimensions } from 'react-nat
 
 interface SliderProps extends ViewProps {
   value?: number;
-  min?: number;
-  max?: number;
+  minimumValue?: number;
+  maximumValue?: number;
   onValueChange?: (value: number) => void;
   disabled?: boolean;
+  minimumTrackTintColor?: string;
+  maximumTrackTintColor?: string;
+  thumbStyle?: any;
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)